### ISSUE TITLE
App crashes due to lack of icons

### ISSUE THREAD

**Issue by AndrewPetrov** [Pyroh/Fluor#35]
App crashes due to lack of icons
http://joxi.ru/12MV5aOfM5RwbA

**Comment by Pyroh** [comment]
Yes it's an issue. But I afraid it'll be fixed once I merge `dev` to `master`...
In the meantime you can either download a [release](https://github.com/Pyroh/Fluor/releases) or clone another branch. If you juste want to use **Fluor** you'd better download a release.
